src/Domain/FunctionsOfTime/Tags.hpp

```diff
@@ -0,0 +1,25 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines tags related to FunctionOfTime quantities
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+#include "Domain/FunctionsOfTime/FunctionOfTime.hpp"
+
+namespace domain {
+namespace Tags {
+/// \ingroup ControlSystemGroup
+/// \ingroup DataBoxTagsGroup
+/// An unordered map of `FunctionOfTime` objects
+struct FunctionsOfTime : db::SimpleTag {
```

Comment:
Thanks for the very quick review!! I have two questions:

1. I made this an action so that evolution executables could have access to the read data. How can I use this in an executable if I make it a free function instead of an action? @nilsdeppe suggested I make this an action, so I'd like him to weigh in on this suggestion

2. @nilsdeppe told me, if I understood him correctly, that his moving mesh code stores the functions of time in an unordered map exactly as I have done here... @nilsdeppe, could you please weigh in on the suggestion to change this tag (and the code) to store individual functions of time in the databox instead of storing them all in an unordered map?

Reply:
1. With the `create_from_options` approach the `FunctionOfTime` also ends up in the DataBox and you can retrieve it from there in evolution executables. The way it gets into the DataBox is different: The option-creation mechanism constructs it instead of the action. You may still use an action to add it to the executable: Make an empty action that has both `initialization_tags` and `initialization_tags_to_keep` set to `tmpl::list<PiecewisePolynomialFromFile>`. If you prefer to have the action load the data instead of the option-creation mechanism (perhaps to have more control over _when_ the data is loaded, not sure if that's relevant) then I still suggest you turn the functionality into a free function and call it from the action.
2. Depends on how the functions of time are going to be used: Do we know at compile time which functions are needed? I.e. can we construct a struct like this:

  ```cpp
  struct CorotationWithBinary {
    constexpr OptionString help = "A function of time for co-rotating the domain with an orbiting binary";
  };
  ```

Reply:
Just leave this as is. It's the right design to work with the moving mesh code.